### ISSUE TITLE
"Commit" and "PR" are obfuscated

### ISSUE THREAD

**Issue by jacob-hd** [telia-oss/github-pr-resource#25]
Instead of showing the actual commit hash and the PR number, random useless characters are being displayed:
![image](https://user-images.githubusercontent.com/31864739/40998131-ad77621a-68bb-11e8-8c11-bd0afb43e467.png)

These almost look like base64 encoded strings... and sure enough, they are:
```
» echo "MDY6Q29tbWl0ODY4Mzg3NzI6YjQ5NTY3ZTg1MDVhMjU0ZmZhNGIxZjM5ZGI2ZjFjZmI3ZmUyMDYxZA==" | base64 --decode
06:Commit86838772:b49567e8505a254ffa4b1f39db6f1cfb7fe2061d
```

```
» echo "MDExOlB1bGxSZXF1ZXN0MTkyNzk2ODMy" | base64 --decode
011:PullRequest192796832
```

Any idea how we can get the actual commit hash and PR number to display instead?

**Comment by itsdalmo** [comment]
Hi and thanks for raising an issue! 

The strings you see there are the Github [Commit ID and PR ID](https://developer.github.com/v4/scalar/id/), which are as you say base64 obfuscated. I just kept them like that because the commit hash and PR number (and more) is output in the metadata on `get` and `put` steps.

That being said, we can always base64 decode and output non-obfuscated strings as well. 
What do you think?

**Comment by billimek** [comment]
+1 for showing the actual PR#.  I don't see a ton of value in seeing something like `pr MDExOlB1bGxSZXF1ZXN0MTkyNzk2ODMy` on the UI.

**Comment by itsdalmo** [comment]
I'll take a look at this in the morning; I believe it will require switching from the GraphQL API for `get` and `put`, but I don't think it will affect cost (rate limit) in any way. And pull request number + commit ref actually means something, so why not 😄

**Comment by itsdalmo** [comment]
@jacob-hd and @billimek - Merged #27 just now and versions are now represented as:

- `pr`: `<number>`
- `commit`: `<sha>`
- `committed`: `<committed date>`

Docker hub should be building a new image as I write this, so the `dev` tag should include the latest changes from master. Would you mind testing the changes and seeing if they work for you?

(PS: Overall it makes the versions more readable as was the goal, and also requires one less API call in a `get` step. The change introduced one more "caveat" which is noted in [this TODO](https://github.com/itsdalmo/github-pr-resource/blob/master/github.go#L212). Think I'll fix that one when someone runs into it and files and issue?).

**Comment by itsdalmo** [comment]
Released in v0.5.0 - closing this issue. Feel free to resubmit if needed.
